common/djangoapps/student/views.py

```diff
@@ -678,7 +680,106 @@ def dashboard(request):
     else:
         redirect_message = ''
 
+    archived_courses = [
```

Comment:
please make a method for getting these details as they are fetched in almost same way for `archived_courses` and `current_courses`